crates/proof-of-sql-parser/src/lib.rs

```diff
@@ -2,6 +2,8 @@
 
 /// Module for handling an intermediate decimal type received from the lexer.
 pub mod intermediate_decimal;
+/// Module for handling an intermediate timestamp type received from the lexer.
+pub mod intermediate_time;
```

Comment:
`time`?

Reply:
time is used in other crates, I think it is better to prefix our use of a time wrapper with someone to differentiate it from say chrono or arrow or data/time in std

Reply:
also core::time

Reply:
I would personally prefer to use naming that is distinct anyways because it makes importing the correct module easier. There is already a module named ```core::time```